src/output_counties.R

```diff
@@ -108,12 +107,12 @@ n_points_per_county_disp_levels <- paste0("'N=", str_trim(format(sort(unique(cou
 county_df_melt[, n_points_per_county_disp := factor(paste0("'N=", str_trim(format(n_points_per_county, big.mark=",",scientific=FALSE)), " Points'"),
                                                     levels=n_points_per_county_disp_levels)]
 
-county_df_melt[variable %like% '_em', estimTypDisp := 'EM']
-county_df_melt[variable %like% '_md', estimTypDisp := 'MD']
-county_df_melt[variable %like% '_naive', estimTypDisp := 'Freq']
+county_df_melt[variable %like% '_em', estimTypDisp := "Max\nLikelihood"]
+county_df_melt[variable %like% '_md', estimTypDisp := "Min\nDist"]
+county_df_melt[variable %like% '_naive', estimTypDisp := "Freq"]
 
 ## Note: the order of the factor levels controls the order along the x-axis in several of the graphs
-county_df_melt[, estimTypDisp := factor(estimTypDisp, levels=c("Freq", "EM", "MD"))]
+county_df_melt[, estimTypDisp := factor(estimTypDisp, levels=c("Freq", "Max\nLikelihood", "Min\nDist"))]
```

Comment:
@rosenbat2, this is more-or-less consistent with the estimator names 
```
estimator_levels <- c("Frequency",
                      "Max Likelihood",
                      "Min Dist", ...
``` 
used in src/fit_embrapa_validation_hmm.R (the Embrapa validation graphs)